features/rsc_can_create_new_hospitals.feature

```diff
@@ -0,0 +1,29 @@
+@javascript
+
+Feature: RSC creates a new hospital 
+    As a RSC,
+    In order to create hospitals,
+    I would like to access the dashboard, click on the “Hospitals” button and fill in the hospital name.
+
+    Background: 
+        Given the following hospitals exists
+            | name        | 
+            | Hospital 2  |
+        Given I am on the landing page
+        And I click on "Hospitals"
+        Then I see "Hospital"
```

Comment:
There's no need for this assertion. We discussed this before. There is no need to assertions in backgrounds